recipes/nginx.rb

```diff
@@ -16,7 +16,11 @@
 #   {
 #     'nginx_status_url' => "https://localhost:82/nginx_status/",
 #     'name' => ['test'],
-#     'ssl_validation' => false
+#     'ssl_validation' => false,
+#     'skip_proxy' => false,
+#     'use_plus_api' => false,
+#     'plus_api_version' => "2"
```

Comment:
To be very nit-picky, this should probably be `2` instead of `"2"` and the end of the line misses a comma. 🙇